src/ui.rs

```diff
@@ -75,10 +76,11 @@ pub fn draw_egui(context: &mut Context, #[cfg(feature = "debug_fps")] fps: f64)
             //     widgets: widget_style,
             //     ..Default::default()
             // });
+
             egui::containers::TopBottomPanel::top("Top menu").show(egui_ctx, |ui| {
                 // egui::containers::CentralPanel::default().show(egui_ctx, |ui| {
                 egui::menu::bar(ui, |ui| {
-                    // ui.set_height(crate::constant::MENU_BAR_SIZE);
+                    ui.set_height(crate::constant::MENU_BAR_SIZE - 4.0);
```

Comment:
why `4.0` ?

Reply:
Egui doesn't create the menu bar with the exact size we specify and seems to take a few more pixels in height. We had fixed this in #62 previously

Reply:
You were right, it should be cleaner now.